tests/test_redirects.py

```diff
@@ -21,50 +21,22 @@ def test_that_checks_redirect_using_incorrect_query_values(self, base_url):
         }
         response = self._head_request(base_url, params=param)
 
-        Assert.equal(
-            response.status_code,
-            requests.codes.not_found,
-            self.response_info_failure_message(
-                base_url,
-                param,
-                response
-            )
-        )
+        assert (response.status_code == requests.codes.not_found,
+                self.response_info_failure_message(base_url, param, response))
 
         parsed_url = urlparse(response.url)
-        Assert.equal(
-            parsed_url.scheme,
-            'http',
-            'Failed by redirected to incorrect scheme %s. \n %s' %
-            (parsed_url.scheme, self.response_info_failure_message(
-                base_url,
-                param,
-                response))
-        )
-        Assert.equal(
-            parsed_url.netloc,
-            urlparse(base_url).netloc,
-            self.response_info_failure_message(
-                base_url,
-                param,
-                response)
-        )
-        Assert.equal(
-            parsed_url.query,
-            urlencode(param),
-            self.response_info_failure_message(
-                base_url,
-                param,
-                response)
-        )
-        Assert.not_equal(
-            self.get_x_backend_server(response),
-            'Unknown',
-            'Failed, x-backend-server was not in the response object. %s' %
-            (self.response_info_failure_message(base_url,
-                param,
-                response))
-        )
+        assert (parsed_url.scheme == 'http', 'Failed by redirected to incorrect scheme %s. \n %s' %
+                (parsed_url.scheme, self.response_info_failure_message(base_url, param, response)))
```

Comment:
We shouldn't need to include the scheme in the message, as pytest will recognise we're doing a comparison and helpfully show both expected and actual values in the traceback.